docs/connectRoutes.md

```diff
@@ -53,107 +55,80 @@ type RouteObject = {
+* **basename** - a prefix that will be prepended to the URL. For example, using a `basename` of `'/playground'`, a route with the path `'/home'` would correspond to the URL path `'/playground/home'`
 
-* **title** - the `title` is similarly the name of the state key for your page title or a selector function for getting it from state. **Redux First Router** will change your page title for you when the route changes, e.g. `document.title = 'foo'`.  As with `location` you can provide a function here which can be useful if you need to implement custom logic to get at the title state. Example: `state => state.get('title')`.
+* **strict** - wether or not a trailing delimiter is allowed when matching path. An url `/foo` will not match a route `FOO: '/foo/'` or an url `/foo/` will not match a route `FOO: '/foo'` when the `strict` option is set to `true`. This is a similar option as the `strict` prop of `<NavLink>` component from `redux-first-router-link` but this option prevents search engines to detect duplicate content (which is bad for SEO) on urls `/foo` and `/foo/`, while the `strict` prop of `<NavLink>` is only meant to determine if the link is active regarding to the current location. Note: search engines may detect duplicate content if somehow a link exists on your site or elsewhere.
 
-* **notFoundPath** - the `notFoundPath` defaults to `'/not-found'`. The address gets redirected here in 2 situations: when you dispatch an action with no matching path, or if you manually call `dispatch(redirect({ type: NOT_FOUND }))`, where `NOT_FOUND` is an export from this package. The type in actions and state will be `NOT_FOUND`, which you can use to show a 404 page. Conversely, if the user visits a URL directly or if you dispatch `NOT_FOUND` without the redirect, the ***current URL is preserved*** but the `NOT_FOUND` type is *also* dispatched, which is the correct way websites are typically supposed to deal with URLs they don't handle. *I.e. just like on Github.com, a 404 graphic will show and the URL stays the same.* So you will *rarely see* `'/not-found'` unless you trigger it (intentionally or by accident). Lastly if you specify `notFoundPath: null`, in the aforementioned scenarios, the URL will display as the previous URL (i.e. the URL currently in the address bar) and fallback to `'/'`, such as in SSR if no history exists yet.
+* **location** - a function to specify where in your Redux state tree **Redux First Router** should expect its reducer to be attached to. This can be omitted if you attach the reducer at `state.location`. You can provide a function here which can be useful if you need to implement custom logic to get at the `location` state. This is especially useful for non-standard state shapes, such as Immutable.js. For example, if `state` is an instance of `Immutable.Map` you might have `state => state.get('location')`.
```

Comment:
I think "a function" should be changed to "A string (a key in the state) or a selector"